### PR TITLE
document our initial devops goals

### PR DESCRIPTION
**Pull request by larsks**

### CODE REVIEW

File: specs/devops-mvp.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+   Using a pull-request workflow gives other members of the operations
+   team the chance to review proposed changes. It also provides us
+   an integration point with continuous integration and automated
+   deployment mechanisms, if we choose to adopt such technology.
+
+   Because we do not wish for this policy to become a roadblock, we
+   may not initially require explicit approval of changes as part of
+   the workflow (although it would be nice to get to that point in the
+   future).
+
```
Comment (naved001) [inline specs/devops-mvp.md:70]: Each item under the `Policy` section starts with `1.` though it does render correctly in the final document (1,2,3).

**Review by naved001 — APPROVED** [review]

File: specs/devops-mvp.md (hunk 1 above)
Comment (larsks) [reply specs/devops-mvp.md:70]: That's correct. I'm taking advantage of markdown syntax: this means items can be re-ordered as necessary and the rendered document will be correct.

Comment (naved001) [reply specs/devops-mvp.md:70]: that's a good idea lol.

**Review by knikolla — APPROVED** [review]
This looks good to me and matches what we discussed in the prior meetings. Thanks for providing references to GitOps/DevOps documentation.

**Pull request merged**